connector_routific/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Routific Connector",
+    "summary": "Connector for Routific Platform",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
```

Comment:
This should change to stable as it's working on customer for a couple of months already, isn't it?